lib/src/widgets/reactions_bottomsheet.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:chatview/src/controller/chat_controller.dart';
+import 'package:chatview/src/models/models.dart';
+import 'package:chatview/src/utils/constants.dart';
+import 'package:flutter/material.dart';
+
+class ReactionsBottomSheet extends StatelessWidget {
+  const ReactionsBottomSheet({
+    Key? key,
+    required this.reaction,
+    required this.chatController,
+    required this.reactionsBottomSheetConfig,
+  }) : super(key: key);
+  final Reaction reaction;
+  final ChatController chatController;
+  final ReactionsBottomSheetConfiguration? reactionsBottomSheetConfig;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: MediaQuery.of(context).size.height * 0.5,
+      color: reactionsBottomSheetConfig?.backgroundColor,
+      child: ListView.builder(
+        padding: reactionsBottomSheetConfig?.bottomSheetPadding ??
+            const EdgeInsets.only(
+              right: 12,
+              left: 12,
+              top: 18,
+            ),
+        itemCount: reaction.reactedUserIds.length,
+        itemBuilder: (_, index) {
+          final reactedUser =
+              chatController.getUserFromId(reaction.reactedUserIds[index]);
+          return Container(
+            margin: const EdgeInsets.only(bottom: 8),
+            padding: const EdgeInsets.all(8),
+            decoration: reactionsBottomSheetConfig?.reactionWidgetDecoration ??
+                BoxDecoration(
+                  color: Colors.white,
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.grey.shade200,
+                      offset: const Offset(0, 20),
+                      blurRadius: 40,
+                    )
+                  ],
+                  borderRadius: BorderRadius.circular(10),
+                ),
+            child: Row(
+              children: [
+                Expanded(
+                  child: Row(
+                    children: [
+                      CircleAvatar(
+                        radius:
+                            reactionsBottomSheetConfig?.profileCircleRadius ??
+                                16,
+                        backgroundImage: NetworkImage(
+                          reactedUser.profilePhoto ?? profileImage,
+                        ),
+                      ),
+                      const SizedBox(width: 12),
+                      Text(
+                        reactedUser.name,
+                        style: reactionsBottomSheetConfig?.reactedUserTextStyle,
+                      ),
+                    ],
+                  ),
+                ),
+                Text(
+                  reaction.reactions[index],
+                  style: TextStyle(
+                    fontSize: reactionsBottomSheetConfig?.reactionSize ?? 14,
+                  ),
+                )
+              ],
+            ),
+          );
+        },
+      ),
+    );
+  }
+
+  static void showBottomSheet({
```

Comment:
Do not use static method. You can show sheet by creating a function in a class and return `showModalBottomSheet` and showModalBottomSheet will return your sheet widget.

Reply:
Resolved